requirements.txt

```
@@ -3,3 +3,4 @@ packaging
 python-daemon
 pyyaml
 six
+importlib-metadata; python_version < "3.10.0"
```

Since the standalone `importlib-metadata` project maintains several different branches that roughly track what's in a given Python stdlib, we should probably pin to a specific branch just for consistent behavior (and also to ensure that it has the features we want). Core does the same thing for the standalone `importlib-resources`- [looking at their compat docs](https://github.com/python/importlib_metadata#compatibility), this should be what we want:

```suggestion
importlib-metadata >= 4.6, < 4.7; python_version < '3.10'
```